src/scripts/affichage.ts

```diff
@@ -26,7 +26,7 @@ export function showElement(element: HTMLElement | null) {
     element?.classList.add('visible')
 }
 
-export function hideSelector(element: HTMLElement, selector: string) {
+export function hideSelector(element: HTMLElement | Document, selector: string) {
```

Comment:
À essayer, mais `Node` doit pouvoir remplacer `HTMLElement | Document` dans beaucoup d'endroits.

Reply:
Ou plus exactement `ParentNode` car c'est sur ce mixin qu'est défini `querySelectorAll`: https://dom.spec.whatwg.org/#ref-for-dom-parentnode-queryselectorall